### DISCUSSION TITLE
Controlled Multiselect Combobox

### DISCUSSION THREAD

**Discussion by tguptatwilio** [twilio-labs/paste#3315] [General] [answered]
Hey Everyone,

I am building an component which uses a combobox with grouped data but it should allow only one selection per group. That means for grouped data, if i have 5 groups 5 items can be selected, one from each group

For example,
If i have 5 groups and i selected value x from group 1, then selecting y value from group 1 should automatically remove value x from the selected values

**Comment by SiTaggart** [comment, marked as answer]
Hi @tguptatwilio 

This is a known limitation of the MultiSelect Combobox right now. It's in our backlog to try and address it but we can't make any promises on timelines, unfortunately. You can follow along here https://twilio-productivity.atlassian.net/browse/DS-52

**Reply by SiTaggart** [reply]
This is now released in Paste core 20.3.0 

Multi select combobox can now be fully controlled by the consumer via inversion of control and using the provided hooks. Documentation can be found here https://paste.twilio.design/components/multiselect-combobox#usemultiselectcombobox-state-hook
